src/shows.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "date": "April 2nd",
+    "venue": "State Park"
+  },
+  {
+    "date": "April 20th",
+    "venue": "Boston College"
+  },
+  {
+    "date": "April 21st",
+    "venue": "Exit Galleries"
+  },
+  {
+    "date": "April 27th",
+    "venue": "Union Tavern"
+  }
+]
```

Comment:
looks good - we should also have who we're playing with and links to it all. but that can be a future ticket.